caas/kubernetes/provider/base_test.go

```diff
@@ -120,7 +120,7 @@ func (s *BaseSuite) setupController(c *gc.C) *gomock.Controller {
 }
 
 func (s *BaseSuite) setupBroker(c *gc.C, ctrl *gomock.Controller, newK8sRestClientFunc provider.NewK8sClientFunc) *gomock.Controller {
-	s.clock = testclock.NewClock(time.Time{})
+	s.clock = testclock.NewClock(time.Now())
```

Comment:
Why?
This smells like something is wrong if we need a time that is close to real-time Now. The *point* of Clock is to make sure you don't depend on real time.

Reply:
ah just changed to Now without thinking because I saw somewhere else uses `time.Now()`, 
yeah , u r right, it could be any time.